x402-proxy-template/src/index.ts

```diff
@@ -149,6 +149,22 @@ app.use("*", async (c, next) => {
 	return fetch(c.req.raw);
 });
 
+/**
+ * Root endpoint - public landing page for the proxy
+ * Used to verify the deployed template is working
+ */
+app.get("/", (c) => {
```

Comment:
The purpose of this proxy is to be transparent unless the path is configured to be protected, OR it's one of the static paths under `/__x402/` for health-check reasons.